.github/workflows/release-artifacts.yml

```diff
@@ -89,17 +89,62 @@ jobs:
           name: debs
           path: debs/*
 
-  build-sdist:
-    name: "Build pypi distribution files"
-    uses: "matrix-org/backend-meta/.github/workflows/packaging.yml@v1"
+  build-wheels:
+    name: Build wheels on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-20.04, macos-10.15]
+        is_pr: [${{ startsWith(github.ref, 'refs/pull/') }}]
+
+        exclude:
+          # Don't build macos wheels on PR CI.
+          - is_pr: true
+            os: "macos-10.15"
```

Comment:
Do we even need wheels for mac? Is the point to make it easier for contributors using macs to make Python-only changes?

Reply:
TBH, I mostly just included mac wheels because cryptography supports them. I don't think its easy to use an existing wheel to bootstrap a source checkout, alas.

Reply:
We write [here](https://matrix-org.github.io/synapse/latest/setup/installation.html?highlight=windows#windows) that

> Running Synapse natively on Windows is not officially supported.

No such wording for [mac](https://matrix-org.github.io/synapse/latest/setup/installation.html?highlight=mac#macos) though.

I guess leave it in since it's only for release builds. :shrug: